<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <widget class="QPushButton" name="networkGameButton">
       <property name="text">
        <string>Сетевая игра</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Тестовый функционал</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
